fendermint/actors/chainmetadata/src/actor.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::Cid;
+use fil_actors_runtime::actor_dispatch;
+use fil_actors_runtime::actor_error;
+use fil_actors_runtime::builtin::singletons::SYSTEM_ACTOR_ADDR;
+use fil_actors_runtime::runtime::{ActorCode, Runtime};
+use fil_actors_runtime::ActorDowncast;
+use fil_actors_runtime::ActorError;
+use fil_actors_runtime::Array;
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::error::ExitCode;
+
+use crate::BlockHash;
+use crate::{ConstructorParams, Method, PushBlockParams, State};
+
+#[cfg(feature = "fil-actor")]
```

Comment:
Maybe this can be applied in the `lib.rs` on top of the whole module, which would deactivate the entire `use fil_actors_runtime` thing too, really no implementation at all.